.claude/skills/internal-image-pull/SKILL.md

```
@@ -0,0 +1,322 @@
+## Constants
+
+- **ECR Registry:** `210086341041.dkr.ecr.us-west-2.amazonaws.com`
+- **ECR Repository:** `ai-dynamo/dynamo`
+- **AWS Account:** `triton-aws` / `210086341041`
+- **AWS Role:** `CS-Engineer-210086341041`
+- **Region:** `us-west-2`
+- **Speedoflight Dashboard:** `http://speedoflight.nvidia.com/dynamo/commits/`
+
+## Image Tag Format
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)